cypress/support/commands.js

```diff
@@ -1,5 +1,38 @@
 /// <reference types="cypress" />
 
-Cypress.Commands.add("dataCy", (value) => {
+Cypress.Commands.add('dataCy', (value) => {
   return cy.get(`[data-cy=${value}]`);
 });
+
+Cypress.Commands.add('interceptGQL', (url, operation, data, alias) => {
```

Comment:
I will need to review this entire file 😿

Reply:
Here is the article where we got this from, does a really awesome job of explaining it and setting it up! 
https://www.jayfreestone.com/writing/stubbing-graphql-cypress/